context.go

```diff
@@ -768,6 +768,15 @@ func (c *Context) IsWebsocket() bool {
 	return false
 }
 
+// Method returns the HTTP method of the requests.
+func (c *Context) Method() string {
+	method := c.Request.Method
+	if method == "" {
+		return http.MethodGet
```

Comment:
why we should return http.MethodGet when c.Request.Method is a empty string?

Reply:
![image](https://user-images.githubusercontent.com/50203211/113678142-e5674580-96f0-11eb-9673-caaa0dda1e33.png)

Because according to the comments of the Method field of the http.Request struct, empty string of Method means GET.

Reply:
Isn't that documentation talking only about "client requests", i.e. requests that you send, not those that you receive?

Reply:
> Isn't that documentation talking only about "client requests", i.e. requests that you send, not those that you receive?

Seems like it's my understanding problem. Thank you for that.